docs/examples/cache.ipynb

```diff
@@ -0,0 +1,297 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3d93c85a-3c9b-4496-990d-8612fcb82fcf",
+   "metadata": {},
+   "source": [
+    "In the examples below we will change the settings multiple times. First we ensure all the changes are temporary and no settings are saved into the configuration file. We also reset the settings to the defaults: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "71214b97-2b64-442d-bd23-98f831b064d0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "settings.auto_save_settings = False\n",
+    "settings.reset()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "df5b047f-a34e-4a70-82d7-b7bb1fcc2fc2",
+   "metadata": {},
+   "source": [
+    "#### User defined cache directory"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "adc31ffe-78a7-4f67-b82a-aa3df81a7e1e",
+   "metadata": {},
+   "source": [
+    "The primary key to control the cache in the settings is **\"cache-policy\"**. The default value is **\"user\"**, which means that the cache is created in the directory defined by the **\"user-cache-directory\"** settings.\n",
+    "\n",
+    "The user cache directory is not cleaned up on exit. So next time you start earthkit.data it will be (probably) there again. Also, when you run multiple sessions of earthkit-data under the same user they will share the same cache. "
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "da6e1e1c-c248-4301-ab89-fea50b73bf57",
+   "metadata": {},
+   "source": [
+    "The settings tells us all the details about the cache policy and location:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "4c2afb82-c630-451f-88e5-f80c8aa404ac",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "user\n",
+      "/var/folders/ng/g0zkhc2s42xbslpsywwp_26m0000gn/T/earthkit-data-cgr\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(settings.get(\"cache-policy\"))\n",
+    "print(settings.get(\"user-cache-directory\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "68e82597-7ef4-40d4-abcb-c3d260a0425e",
+   "metadata": {},
+   "source": [
+    "The path to the current cache directory can also be queried through the **cache** object:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "44583d41-f1d3-4c28-8af5-90d659cdae31",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'/var/folders/ng/g0zkhc2s42xbslpsywwp_26m0000gn/T/earthkit-data-cgr'"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cache.cache_directory()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f6665b3b-5176-4161-9671-b20eb3bcc483",
+   "metadata": {},
+   "source": [
+    "We are free to change the user cache directory to another path:"
```

Comment:
> I think it would be good to state whether the old cache dir is copied to the new location or not.

I agree. The cache content is not copied. Actually, to change the cache directory like that is not perfectly safe because there can be downloads (e.g. mars) still active using the old cache! Maybe the whole notebook contents should be reconsidered.

Reply:
Indeed, I think it's best not to try to move the directory, it's just good to clarify in the text what happens.